keyboards/kbd75/keymaps/kingwangwong/keymap.c

```diff
@@ -0,0 +1,148 @@
+#include QMK_KEYBOARD_H
+
+#define _QW 0
```

Comment:
You can convert all your layer defines into enums so that they will get numbered automatically:

```c
enum layer_names {
    _QW,
    _FO,
    _FL,
    _CL,
    _DL
};
```
etc.